<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>page_2</class>
 <widget class="QWidget" name="page_2">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>480</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QPushButton" name="pBack_page2">
   <property name="geometry">
    <rect>
     <x>650</x>
     <y>330</y>
     <width>150</width>
     <height>150</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="Resources.qrc">
     <normaloff>:/new/prefix1/arrows_back_icon.png</normaloff>:/new/prefix1/arrows_back_icon.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>90</width>
     <height>90</height>
    </size>
   </property>
   <property name="flat">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QWidget" name="horizontalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>60</y>
     <width>571</width>
     <height>361</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QLabel" name="label">
      <property name="styleSheet">
       <string notr="true">border-radius: 35px; 
padding: 40px;
background-color: rgba(113, 160, 227);
color:white;</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item>
     <spacer name="horizontalSpacer">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeType">
       <enum>QSizePolicy::Fixed</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <widget class="QLabel" name="label_2">
      <property name="styleSheet">
       <string notr="true">border-radius: 35px; 
padding: 40px;
background-color: rgba(176, 204, 239);
color:white;</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item>
     <spacer name="horizontalSpacer_2">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeType">
       <enum>QSizePolicy::Fixed</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <widget class="QLabel" name="label_3">
      <property name="styleSheet">
       <string notr="true">border-radius: 35px; 
padding: 40px;
background-color: rgba(200, 218, 239);
color:black;</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>70</y>
     <width>111</width>
     <height>221</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>DearMyday</family>
     <pointsize>11</pointsize>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <property name="contextMenuPolicy">
    <enum>Qt::DefaultContextMenu</enum>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="Resources.qrc">
     <normaloff>:/new/prefix1/CLOCK_IMG.png</normaloff>:/new/prefix1/CLOCK_IMG.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>100</width>
     <height>100</height>
    </size>
   </property>
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="autoRepeat">
    <bool>false</bool>
   </property>
   <property name="autoDefault">
    <bool>false</bool>
   </property>
   <property name="flat">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="pDB">
   <property name="geometry">
    <rect>
     <x>550</x>
     <y>80</y>
     <width>121</width>
     <height>221</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="Resources.qrc">
     <normaloff>:/new/prefix1/database.png</normaloff>:/new/prefix1/database.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>100</width>
     <height>100</height>
    </size>
   </property>
   <property name="flat">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="pToggle">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>70</y>
     <width>151</width>
     <height>241</height>
    </rect>
   </property>
   <property name="maximumSize">
    <size>
     <width>16777215</width>
     <height>16777215</height>
    </size>
   </property>
   <property name="contextMenuPolicy">
    <enum>Qt::DefaultContextMenu</enum>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="Resources.qrc">
     <normaloff>:/new/prefix1/OFF_IMG.png</normaloff>
     <normalon>:/new/prefix1/ON_IMG.png</normalon>:/new/prefix1/OFF_IMG.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>100</width>
     <height>100</height>
    </size>
   </property>
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>false</bool>
   </property>
   <property name="autoDefault">
    <bool>false</bool>
   </property>
   <property name="default">
    <bool>false</bool>
   </property>
   <property name="flat">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="on_off_label">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>340</y>
     <width>91</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Aa삐삐롱</family>
     <pointsize>28</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="Alarm_label">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>330</y>
     <width>141</width>
     <height>71</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Aa삐삐롱</family>
     <pointsize>28</pointsize>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string>Alarm</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="Data_label">
   <property name="geometry">
    <rect>
     <x>530</x>
     <y>330</y>
     <width>150</width>
     <height>80</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Aa삐삐롱</family>
     <pointsize>28</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string>Data</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>50</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Aa삐삐롱</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">padding: 40px;
background-color: rgba(89, 141, 204);
color:white;</string>
   </property>
   <property name="text">
    <string>Setting</string>
   </property>
  </widget>
  <zorder>pBack_page2</zorder>
  <zorder>horizontalLayoutWidget</zorder>
  <zorder>pDB</zorder>
  <zorder>pushButton</zorder>
  <zorder>pToggle</zorder>
  <zorder>on_off_label</zorder>
  <zorder>Alarm_label</zorder>
  <zorder>Data_label</zorder>
  <zorder>label_4</zorder>
 </widget>
 <resources>
  <include location="Resources.qrc"/>
 </resources>
 <connections/>
</ui>
